components/family-studies/ui/src/main/resources/PhenoTips/FamilySelector.xml

```diff
@@ -144,15 +144,15 @@
             this.createFamily.observe('click', this._onOptionSelected.bind(this) );
             this.assignFamily.observe('click', this._onOptionSelected.bind(this) );
 
-            createLabel.insert(this.createFamily).insert("Create a new family");
+            createLabel.insert(this.createFamily).insert("$services.localization.render('phenotips.UIXField.family_membership.create')");
```

Comment:
Whenever you have to modify a script and notice that doesn't have escaping, definitely yes. If you want to actually go through all the scripts and add escaping now, that would be awesome, but I can imagine it would take a while.